src/Bonus.java

```diff
@@ -38,11 +49,18 @@ public static void main(String[] args) {
         - Schrijf vervolgens een for-loop om de items in de hashset een voor een aan de String variabele toe te voegen.
         - Return de (gevulde) String variabele
          */
+        StringBuilder convert = new StringBuilder();
+        for (Integer i : secretnumber) {
+        convert.append(i);
+
+        }
+
+        return convert;
     }
 
 
 
-    public static void/*moet dit "void" zijn of wat anders?*/ feedback(String stringnumber) {
+    public static void feedback(String stringnumber) {
```

Comment:
Type void is prima gezien je enkel een regel tekst print mocht deze methode aangeroepen worden.